app/Models/User.php

```diff
@@ -64,7 +61,8 @@ class User extends Authenticatable implements Camelizable
      */
     protected $fillable = [
         'name',
-        'is_advertiser', 'is_publisher',
+        'is_advertiser',
+        'is_publisher',
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "is_publisher" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=php%3AS1192)